.gitignore

```diff
@@ -30,4 +30,7 @@ packages/*/dist/
 
 # Generated .env
 .env
-**.env
\ No newline at end of file
+**.env
```

Comment:
Out of curiosity, what do the `**.` patterns match? 

That's the first time I see it being used before a dot like this

Reply:
AFAIK ** is anything including empty. * is anything except empty